app/lib/actions/pulp/repository/presenters/docker_presenter.rb

```diff
@@ -31,7 +31,7 @@ def humanized_details
 
             if content_started?(download_details)
               if items_total(download_details) > 0
-                ret << (_("New images: %{count}.") % {:count => count_summary})
+                ret << (_("New blobs: %{count}.") % {:count => count_summary})
```

Comment:
@jlsherrill I'm not an v2 api expert, but I think the blob is the only part that gets updated as the task progresses. The tags and manifests look like they are more akin to distributions in yum, in that they're metadata that get converted to units vs downloaded units.